### PR TITLE
Create elora.json

### PR DESCRIPTION
**Pull request by elorahub**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/elora.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "elorahub",
+        "email": "elorahubph@gmail.com"
+    },
+    "record": {
+        "A": "162.159.205.11",
+        "CNAME": "apex-loadbalancer.netlify.com"
```
Comment (Stef-00012) [inline domains/elora.json:8]: You can not use CNAME with other records

**Review by Stef-DP — DISMISSED** [review]

**Review by elorahub — COMMENTED** [review]
Done changing, sorry I'm still a newbie

File: domains/elora.json (hunk 1 above)
Comment (elorahub) [reply domains/elora.json]: Hello I already done updating, thank you inadvance

**Comment by elorahub** [comment]
> This site is inaccessible, please provide a preview.

https://rpwradio.netlify.app/ preview cname is from netlify.

thank you

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**